scripts/factory/desktop/burnavalon7_761_firmware.desktop

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=false
+Icon[en_US]=gnome-panel-launcher
+Exec=gnome-terminal -e '/home/factory/Avalon-extras/scripts/factory/burn-761firmware.sh 761'
```

Comment:
burn-761firmware.sh is a very clear name, so it is little confused use '761' as a parameter.